WindowGUISample pull request

dotoritos-kim opened the pull request:
This example shows how to use "TrackBar" and "SetMouseCallback". Thank you for your positive review.

dotoritos-kim committed:
WindowGUISample

shimat commented on SamplesCS/ISample.cs:
```diff
@@ -2,6 +2,7 @@
 using System.Collections.Generic;
 using System.Linq;
 using System.Text;
+using static SamplesCS.Samples.WindowGUISample;
```
This seems to be unnecessary.

dotoritos-kim replied:
I have accepted your advice. Thank you

shimat commented on CameraOpenCV/CameraOpenCV.csproj:
```diff
@@ -11,8 +11,8 @@
-    <TargetPlatformVersion Condition=" '$(TargetPlatformVersion)' == '' ">10.0.18362.0</TargetPlatformVersion>
-    <TargetPlatformMinVersion>10.0.18362.0</TargetPlatformMinVersion>
+    <TargetPlatformVersion Condition=" '$(TargetPlatformVersion)' == '' ">10.0.17763.0</TargetPlatformVersion>
+    <TargetPlatformMinVersion>10.0.17763.0</TargetPlatformMinVersion>
```
🤔

dotoritos-kim committed:
Fix unnecessary

dotoritos-kim committed:
Fix encoding error

? -> @

dotoritos-kim committed:
Unnecessary code commenting out

dotoritos-kim committed:
PerspectiveTransformSample Add

shimat commented on SamplesCS/Samples/PerspectiveTransformSample.cs:
```diff
@@ -0,0 +1,60 @@
+                point2Fs.Add(new Point2f(x, y));
+                if (point2Fs.Count == 4)
+                {
+                    srcPoints = new Point2f[]
```
`srcPoints = point2Fs.ToArray();`

dotoritos-kim replied:
Thank you for not thinking

shimat commented on SamplesCS/Samples/WindowGUISample.cs:
```diff
@@ -0,0 +1,173 @@
+                {
+                    var n = pos - maxIterations;
+                    var an = n > 0 ? n : -n;
+                    var element = Cv2.GetStructuringElement(
```
using var element = ...

dotoritos-kim committed:
https://github.com/shimat/opencvsharp_samples/pull/16#discussion_r361397084
https://github.com/shimat/opencvsharp_samples/pull/16#discussion_r361397613
Fixed these two advice.

shimat reviewed (approved):
Thanks!
